core/src/main/res/drawable/ic_paperclip.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="12dp"
+    android:width="12dp"
+    android:viewportWidth="12"
+    android:viewportHeight="12">
+
+    <group
+        android:scaleX="0.5"
```

Comment:
Why is this needed? Can't you just adapt the viewport to scale it? That simplifies the icon

Reply:
Apologies for the delay on this. Was away for a while and then took another while to get caught up.

AFAICT the viewport attributes only change the amount of the icon that is visible, ie they crop it and setting it to 6/6 ends up only showing the top left of the paperclip.

I had a look around online and couldn't see any obvious other solution short of halving all of the numbers in the source icon pathData attribute.

Reply:
I would have removed the scale and set the viewport to 24/24, not to 6/6

Reply:
@caoilte does setting the viewport to 24/24 and removing the scaling work?

Reply:
aaaah. neat! yeah that did the trick! thanks for explaining.